### ISSUE TITLE
No Uploads possible

### ISSUE THREAD

**Issue by kshb210** [FilenCloudDienste/filen-mobile#254]
I cannot, for example, upload from Lightroom at this time. I go the usual way in Android via "Share" and select Filen. Filen then also opens, I can select the destination folder and click "Upload". But then nothing happens.

So I can only start uploads from the file manager in Android or in Filen itself.

This happens with all external apps, as well as on my Galaxy S21 and on the Nothing Phone 2.

**Comment by DanielProg39** [comment]
Yeah, uploading using share feature is bugged for a while now. Devs should consider to investigate it

**Comment by juan-vilas** [comment]
Same issue here

**Comment by auditum-mpa** [comment]
Have the same here.

**Comment by shawn1thompson** [comment]
I'm not sure if this is the same issue, but on Android I only have upload files and no upload folder. It won't let me select a folder when I try upload files. I've tried initiating from the file manager, but there is no share option for me.

**Comment by clutwo** [comment]
Sadly same issue.

**Comment by Dwynr** [comment]
Seems to be an issue with the react-native-share-menu lib we are using for android share intents. I will for it as soon as I can and fix it.

**Comment by auditum-mpa** [comment]
> Seems to be an issue with the react-native-share-menu lib we are using for android share intents. I will for it as soon as I can and fix it.

Thanks for the good job!

**Comment by nar3n12** [comment]
I'm also facing the same issue. Let me know how I can help, maybe a log file woiuld help? Or the issue is already identified and being fixed?

**Comment by auditum-mpa** [comment]
Sehr gute Artbeit. Vielen Dank
